### DISCUSSION TITLE
USGWI Server not started?

### DISCUSSION THREAD

**Discussion by Mum-O-Killowe** [iiab/iiab#3831] [General]
I think I may have screwed up.   I was trying to shut off the SSID broadcasting of the "Internet-In-A-Box" and when I rebooted I get this.  
![iiab_Issue](https://github.com/user-attachments/assets/8c59cf94-f6a1-44f7-af71-d9f9ec521e15)

I can still access the iiab user page but I can't seem to log back in to fix this as the error pop up keeps me from gettign to the login.

Any thoughts?

**Comment by holta** [comment]
Hi @Mum-O-Killowe !

Can you please run `sudo iiab-diagnostics` and reply here with the URL it generates?

**Reply by Mum-O-Killowe** [reply]
https://paste.centos.org/view/32ec7b97


Sorry, posted the wrong thing the first time.  I'm a total noob.   lol  (as if it didn't show)

Thanks

**Comment by tim-moody** [comment]
I see a lot of things working in Sept and then problems, including hostapd doesn't start, beginning Oct 11, so we probably need to know what happened on that date.

But also

>Error: Internet is Required for this Command, but Not Available. INST-ZIMS

Seems IIAB is not connected to the internet.

**Reply by Mum-O-Killowe** [reply]
I think it was something I did to try to shut off the dual use wireless network and an attempt to hide the SSID.   I'm not 100% certain.
It is currently plugged in directly to my Internet router.  Could the Ethernet port be shut off somehow?

**Reply by Mum-O-Killowe** [reply]
No one has any  ideas on a file I might be able to edit right off the SD Card to reenable the wifi and get this working?

**Comment by tim-moody** [comment]
Not much to go on. You could try a reinstall which will preserve any existing content:

as root connected via ssh and with IIAB having an internet connection. (there are no guarantees)

```
cd /opt/iiab/iiab
git pull
./iiab-install --reinstall

cd /opt/iiab/iiab-admin-console
git pull
./install 

```

**Reply by Mum-O-Killowe** [reply]
Perfect, I'll try that.   Thanks,
